CHANGES.rst

```diff
@@ -6,7 +6,8 @@ There's a frood who really knows where his towel is.
 1.1b2 (unreleased)
 ^^^^^^^^^^^^^^^^^^
 
-- Nothing changed yet.
+- Remove unnecessary adapter (closes`621`_).
```

Comment:
@rodfersou I think this reference is unnecessary; also, please write a better changelog entry.